packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -15,13 +15,27 @@ export const ChatList = (props: MgtTemplateProps & IChatListItemInteractionProps
   // TODO: change this to use StatefulGraphChatListClient
   const chatClient: StatefulGraphChatClient = useGraphChatClient('');
   const [chatState, setChatState] = useState(chatClient.getState());
+  const [selectedItem, setSelectedItem] = useState<string>();
+  const [readItems, setReadItems] = useState<string[]>([]);
```

Comment:
ChatList managing the selected item makes sense, but why would it be responsible for whether items are read or not? Couldn't they manage that on their own?

Reply:
It was a challenge trying to give ChatListItem full control over the read so I gave up on it.  This worked for me. Also, the soon to be MarkAllAsRead functionality is a responsibility of the ChatList, so, I figured this would support that. I can try again though if you're strongly opinionated about this.

Reply:
I am pretty opinionated about this, but I can look into it a bit and get back to you. It seems like it should be possible and would be easier conceptually.

Reply:
I think I figured it out. I'll update this.